loops-arrays-objects.js

```diff
@@ -131,5 +168,62 @@ const students = [
 // ]
 // ==========================================
 
+//without use of variables
+// for (let i = 0; i < students.length; i++) {
+//     switch (true) {
+//         case (students[i].zipCode === '3513'):
+//             students[i].neighborhood = 'Pijlsweerd';
+//             break;
+//         case (students[i].zipCode === '3514'):
+//             students[i].neighborhood = 'Vogelenbuurt';
+//             break;
+//         case (students[i].zipCode === '3512'):
+//             students[i].neighborhood = 'Binnenstad';
+//             break;
+//         case (students[i].zipCode === '3531'):
+//             students[i].neighborhood = 'Lombok';
+//             break;
+//         case (students[i].zipCode === '3572'):
+//             students[i].neighborhood = 'Wittevrouwen';
+//             break;
+//         case (students[i].zipCode === '3581'):
+//             students[i].neighborhood = 'Oudwijk';
+//             break;
+//         case (students[i].zipCode === '3583'):
+//             students[i].neighborhood = 'Schildersbuurt';
+//             break;
+//     }
+// }
+// console.log(students)
+
+//with use of variables
+for (let i = 0; i < students.length; i++) {
+    let neighborhood = students[i].neighborhood
+    let zip = students[i].zipCode
+    switch (true) {
```

Comment:
je had hier ipv true ook zip variable kunnen zetten en dan kan je bij case tussen haakjes alleen de zipcode zetten. is wat korter